express/blocks/link-list/link-list.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Adobe. All rights reserved.
+ * Copyright 2022 Adobe. All rights reserved.
```

Comment:
```suggestion
 * Copyright 2021 Adobe. All rights reserved.
```
Sorry for the confusion: I meant for the copyright year to be 2022 for the newly added blocks only, not existing ones...